scapy/layers/tls/session.py

```diff
@@ -558,98 +558,113 @@ def compute_sslv2_km_and_derive_keys(self):
 
     # Secrets management for TLS 1.3
 
-    def compute_tls13_early_secrets(self):
+    def compute_tls13_early_secrets(self, external=False):
```

Comment:
Could you explain what is the external argument in the docstring?

Reply:
Yes, different labels are used in the key schedule according it's a resumption or an external PSK. 
I will explain that in the docstring